hatch_build.py

```diff
@@ -267,6 +267,7 @@
         "pytest-mock>=3.12.0",
         "pytest-rerunfailures>=13.0",
         "pytest-timeouts>=1.2.1",
+        "pytest-unordered>=0.6.1",
```

Comment:
Sdk dev deps should go on its pyproject.toml I think

Reply:
Ah yeah, good call.  I think we can add it in both though, that's a nice little tool for such cases (wiht no other dependencies than pytest)

Reply:
Added to both now.